components/tools/OmeroJava/test/integration/DeleteServiceFilesTest.java

```diff
@@ -359,9 +358,7 @@ RepositoryPrx getLegacyRepository() throws Exception {
                     + desc.getName().getValue();
             s += "\nFound repository:" + desc.getPath().getValue()
                     + desc.getName().getValue();
-            if (FilenameUtils.equals(
-                    FilenameUtils.normalizeNoEndSeparator(dataDir),
-                    FilenameUtils.normalizeNoEndSeparator(repoPath))) {
+            if (removeEndSeparator(dataDir).equals(removeEndSeparator(repoPath))) {
```

Comment:
Note that `FilenameUtils.normalizeNotEndSeparator()` does a lot more than just strip a trailing path separator. Are we sure that's all that matters here? I think you'd be safest with `Paths.get(...).normalize()`, which will also strip a trailing path separator if present.